app.js

```diff
@@ -58,11 +58,32 @@ const onCreateEditButton = (inputElement) => {
     return editButton;
 }
 
+const handleDone = (todo, inputElement) => {
+    todo.done = !todo.done;
+    inputElement.classList.toggle('completed', todo.done);
+}
```

Comment:
As `done` is not toggleable let's not make it confusing.

```suggestion
const handleDone = (todo, inputElement) => {
    todo.done = true;
    inputElement.classList.toggle('completed', todo.done);
}
```